charts/k8tz/templates/admission-webhook.yaml

```diff
@@ -31,6 +31,12 @@ webhooks:
       - key: k8tz.io/controller-namespace
         operator: NotIn
         values: ["true"]
+      {{- if gt (len .Values.webhook.ignoredNamespaces) 0 }}
```

Comment:
what about `{{- if .Values.webhook.ignoredNamespaces }}`?
it feels more readable, and will not throw error if value is null (instead of an empty array)

Reply:
I replaced it